osbs/cli/main.py

```diff
@@ -625,7 +625,10 @@ def cli():
 
     get_login_parser = subparsers.add_parser(str_on_2_unicode_on_3('login'),
                                              help='perform login and store token for later use')
-    get_login_parser.add_argument('TOKEN', help='token to be used for login')
+    get_login_parser.add_argument('--token', help='token to be used for login '
+                                                  '(takes precedence over username and password)', action="store")
```

Comment:
it isn't possible to create exclusive groups between 2 and 1 parameters

I wrote in there that it takes precedence but I can also just make it error if both variants are provided